client/app/scripts/superdesk-publish/filters/view/production-test-view.html

```diff
@@ -0,0 +1,51 @@
+<section class="main-section.filter">
+    <div class="preview-layout" ng-class="{closed: !selected.preview}">
+        <div class="list-pane">
+            <header>
+                <div class="sortbar pull-left">
+                    <span ng-repeat="type in resultType">
+                        <input type="radio" ng-click="fetchResults()" name="type" value="{{type.value}}" data-ng-model="model.selectedType"> {{type.id}}
+                    </span>
+                </div>
+
+            </header>
+            <div class="content">
+                <div sd-shadow>
+                    <table class="table styled-table1">
+                        <thead>
+                            <tr>
+                                <th></th>
+                                <th translate>State</th>
+                                <th translate>Unique Name</th>
+                                <th translate>Headline</th>
+                                <th translate>Keywords</th>
+                                <th translate>Source</th>
+                                <th translate>ANPA Category</th>
+                                <th translate>View</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr ng-class="{active: result_item === selectedItem, archived: result_item.archived || result_item.created, locked: lock.isLocked}" tabindex="1" ng-keyup="handleKeyEvent($event)" ng-repeat="result_item in testResult track by result_item._id" ng-click="preview(result_item);">
+                                <td><i class="filetype-icon-{{result_item.type}}" ng-hide="hover || item.selected"></i></td>
+                                <td><div class="state-label state-{{result_item.state}}">{{ result_item.state}}</div></td>
+                                <td>{{ result_item.unique_name}}</td>
+                                <td>{{ result_item.headline | limitTo: 60}}
+                                    <span sd-item-container data-item="result_item" title="{{ result_item.container }}" 
+                                    class="container"></span>
+                                </td>
+                                <td>{{ result_item.slugline | limitTo: 40}}</td>
+                                <td>{{ result_item.source}}</td>
+                                <td>{{ result_item['anpa-category'].name }}</td>
```

Comment:
Please check if any of these result_item's properties do not change and could be one-time bound, reducing the number of `$watches`.